tests/acceptance/17_packages/12_new/timed/packagesmatching.cf

```diff
@@ -21,6 +21,8 @@ bundle agent init
 
   files:
     test_pass_1::
+      "$(sys.workdir)/modules/packages/."
```

Comment:
this is probably not needed now that I am including modules in testall `--includes`